src/app/feed/info-box/info-box.component.ts

```diff
@@ -109,18 +105,29 @@ export class InfoBoxComponent implements OnInit, OnChanges {
 
 	}
 	sortMentions(statistics) {
-		let sortable = [];
+		let stored = [];
 		if (statistics !== undefined && statistics.length !== 0) {
 			for (const s in statistics) {
 				if (s) {
-					sortable.push([s, statistics[s]]);
+					for (let i = 0; i < statistics[s][0].length; i++) {
+						stored.push(statistics[s][0][i]);
+					}
 				}
 			}
-			sortable.sort(function (a, b) {
-				return b[1] - a[1];
-			});
-			sortable = (sortable.slice(0, 10));
-			this.topMentions = sortable;
+			stored = stored.reduce(function (acc, curr) {
+				if (typeof acc[curr] === 'undefined') {
+						acc[curr] = 1;
+				} else {
+					acc[curr] += 1;
+				}
+				return acc;
+			}, []);
+			this.topMentions = Object.keys(stored)
+			.map(key => key.trim())
+			.filter(key => key !== '')
+			.map(key => ([key, stored[key]]))
+			.sort((a, b) => b[1] - a[1]);
+			this.topMentions = this.topMentions.slice(0, 10);
```

Comment:
The slice can still go with the previous chained operation. Also, there is one level of indentation missing.

Reply:
Yes, I am fixing it.